examples/Devices/Keyboardio/Model100/Model100.ino

```diff
@@ -306,6 +306,33 @@ KEYMAPS(
    ___)
 ) // KEYMAPS(
 
+PALETTE(
```

Comment:
I think it probably makes sense to create "red" and "unset" colors, to make this readable and nice looking.

Reply:
For sure!